### ISSUE TITLE
barebone quick start" recommendations?

### ISSUE THREAD

**Issue by smblee** [UnlyEd/next-right-now#151]
Hello!

I am trying to dip my toe into NextJS and I tend to search for templates to see what "best" practices are out there for a certain framework I am interested in.

This is definitely one of the most fully featured, production-ready templates, but it has definitely been hard to rip out things I don't need and get started fresh so I can actually develop my own app. 

I don't find the "plugin" system that highly customizable since "plugins" seem to already be bundled into the 2 main templates you offer, and I have been having hard time trying to rip things out that I don't need (i18n/locize, example app, MST tenancy (although looks like I can opt out by not using certain now json files), and etc.). I was hoping the system would be more pick-and-choose-y.

Did I miss some documentation on these steps? Any guidance on this would be helpful!

**Comment by Vadorequest** [comment]
Hi @smblee and thanks for your feedback.

I'm aware the current available preset may contain "too much" for some people. It's my intent to make some lighter presets as mentioned in https://unlyed.github.io/next-right-now/concepts/presets#which-presets-are-being-considered. But it's really a lot of work as far as maintaining goes, once the presets are released, to keep them up to date. That's why I haven't done it yet (kinda trying to make as good as possible first before creating many presets, to avoid having to maintain them all), even though it's kinda utopian. 

So, for now, the only thing that might help you is the documentation about "how to remove stuff", which is a bit light, such as:
- I18n: https://unlyed.github.io/next-right-now/guides/i18n/remove-i18n
- Example app: Just remove all files under /docs folder, or just change the index.tsx file to not use the docs/index.tsx and make your own app there. (docs won't be removed, but won't be used anymore unless going explicitly to the url) 
- MST tenancy: There is "no way" to remove it (it's really a ton of work, across the whole app, very complicated). My suggestion would be to keep it and use one tenant only for now, that's clearly the easiest way to get started, regarding MST in particular.

Feel free to open a PR regarding the documentation site, if you'd have any findings you'd like to share with the community to guide how to remove a particular "plugin/feature".

Unfortunately, it'd make the whole thing much harder to maintain if it had to embed a "more pick-and-choose-y" tool because each plugin/feature might have a lot of impact on many parts of the source code. That's why "presets" exist in the first place, to pick a preset that match your needs best. It's just that there are only a few options at the moment (2).

**Comment by Vadorequest** [comment]
@smblee I've used your feedback to improve the documentation. PR is in progress at https://github.com/UnlyEd/next-right-now/pull/160

Feel free to give me your feedback.

I'll close this issue once the PR is merged. :)

**Comment by Vadorequest** [comment]
I'm merging the PR now, but feel free to comment it or contribute directly through another PR.

**Comment by Vadorequest** [comment]
Related doc:
- https://unlyed.github.io/next-right-now/guides/demo-examples/
- https://unlyed.github.io/next-right-now/guides/tenancy/
- https://unlyed.github.io/next-right-now/guides/tenancy/remove-MST.html
- https://unlyed.github.io/next-right-now/concepts/tenancy
- https://unlyed.github.io/next-right-now/faq

**Comment by smblee** [comment]
Great job! Appreciate the effort that you took to improve the documentation 👍

**Comment by Vadorequest** [comment]
I hope it helps answering your questions, and will help future newcomers :)
